ci/scripts/get-git-sha-for-release-tag.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const { Octokit } = require('@octokit/rest');
+const octokit = new Octokit()
+
+const run = async function run() {
```

Comment:
after consultation with Steven Fountain esq, we decided assigning to variable here is a bit OTT, so suggest
```suggestion
async function run () {
```